### PR TITLE
Integrate Post Creation Backend to Newsfeed UI

### PR DESCRIPTION
**Pull request by XPERIA679**
**OVERVIEW**
Integrate creating posts backend to newsfeed with UI

**SNIPPETS**
image

### CODE REVIEW

**Comment by XPERIA679** [comment]
**OVERVIEW**
Added Image Preview for File Upload

**SNIPPETS**
image

File: src/app/Http/Controllers/UserPostController.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ public function __construct() {
     public function create(CreateUserPostRequest $request): RedirectResponse
     {
         $this->userPostService->create($request);
-        return redirect('/');
+        return redirect('/posts-page');
     }
```
Comment (janrebpol) [inline src/app/Http/Controllers/UserPostController.php:31]: what is this

File: src/public/js/fileHandler.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        const reader = new FileReader();
+        reader.onload = function() {
+            previewContainer.innerHTML = `<img src="${reader.result}" class="w-full h-full object-cover">`;
+            previewContainer.classList.remove('hidden');
+        }
+        reader.readAsDataURL(file);
+    } else {
+        previewContainer.innerHTML = '';
+        previewContainer.classList.add('hidden');
+    }
```
Comment (hipe01mark) [inline src/public/js/fileHandler.js]: Prevent else in the if statement!!!!!!!!!!

**Review by hipe01mark — APPROVED** [review]
LGTM

**Pull request merged**